### PR TITLE
[ALS-5755] Switch time series processor to ISO timestamps

### PR DESCRIPTION
**Pull request by Luke-Sikina**
- Make service that does this
- Isolate time series logic a bit more
- Tests

### CODE REVIEW

File: processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+@Service
+public class TimeSeriesConversionService {
+
+    public String toISOString(Long unixTimeStamp, TimeZone timeZone) {
+        Date date = new Date(unixTimeStamp);
+        SimpleDateFormat isoFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.S'Z'");
```
Comment (ramari16) [inline processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java:14]: Is this different than `DateTimeFormatter.ISO_INSTANT`, and if so, why is that required?

Comment (Luke-Sikina) [reply processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java]: That works. Switched.

File: processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.stereotype.Service;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+@Service
+public class TimeSeriesConversionService {
+
+    public String toISOString(Long unixTimeStamp, TimeZone timeZone) {
```
[the remark this answers is not in the document]
Comment (Luke-Sikina) [reply processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java]: It's a Unix timestamp; they're UTC or we're not following the standard correctly.

[the remark this answers is not in the document]
Comment (Luke-Sikina) [reply processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java]: I added the timezone to be explicit about the output

File: processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+import java.text.SimpleDateFormat;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.util.TimeZone;
+
+@Service
+public class TimeSeriesConversionService {
+
+    public String toISOString(Long unixTimeStamp) {
```
Comment (ramari16) [inline processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java:13]: Do we actually want to do this -- we are losing information, the person requesting the data may be better off just getting the unix time stamp, no?

File: processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (ramari16) [reply processing/src/main/java/edu/harvard/hms/dbmi/avillach/hpds/processing/timeseries/TimeSeriesConversionService.java]: I was kind of asking rhetorically, sometimes these are when the data was processed, sometimes they are when a medication was given or a test result. Which loses meaning if you don't know the local time. I'm guessing 99% of our data is in 4 time zones so currently it's not much of an issue
